pkg/kv/kvserver/store_raft.go

```diff
@@ -320,6 +321,54 @@ func (s *Store) HandleRaftUncoalescedRequest(
 	return enqueue
 }
 
+// checkAndUpdateCrossLocalityRaftMetrics updates store metrics associated with
+// cross-locality raft messages. Cross-region metrics monitor activities across
+// different regions. Cross-zone metrics monitor cross-zone activities within
+// the same region or in cases where region tiers are not configured.
+func (s *Store) checkAndUpdateCrossLocalityRaftMetrics(
+	ctx context.Context,
+	fromReplica roachpb.ReplicaDescriptor,
+	toReplica roachpb.ReplicaDescriptor,
+	inc int64,
```

Comment:
`msgSize` is a better name because then the caller knows what we're measuring here. I know, easy enough to infer, but clearer is better.

Reply:
Done.